graphql_server/server.go

```diff
@@ -40,8 +42,13 @@ func (s *Server) registerEchoMutation(schema *schemabuilder.Schema) {
 func (s *Server) registerGame(schema *schemabuilder.Schema) {
 	queryObj := schema.Query()
 	queryObj.FieldFunc("games", func(ctx context.Context, args struct{ TitleRegex *string }) ([]*data.Game, error) {
+		allGames, err := s.SqlDriver.GetAllGames(ctx)
```

Comment:
This is a bit cheeky, I could probably pass the regex down, but this proves the point enough and changes less code 🤷‍♀️